### ISSUE TITLE
Custom datagrid filters appear to interfere with each other

### ISSUE THREAD

**Issue by sjcetx** [vmware/clarity#3824]
When multiple custom datagrid filters are added some will stop working even though they all work when added individually.

https://stackblitz.com/edit/clarity-light-theme-v2-n8b4es

In the above example if both columns have a filter added only the key filter will work, the `accepts` method in the value filter is never called.  Removing the custom filter from the key column will allow the value column filter to work. 

* Angular: 8
* Node: 10.16.3
* Clarity: 2

**Comment by amellnik** [comment]
Probably related: https://github.com/vmware/clarity/issues/3486

**Comment by sjcetx** [comment]
@amellnik It does appear to be, I tried adding in a third column and it worked fine while the second didn't as in #3486.  
In my my project I do have a Datagrid with 4 columns, all with custom filters and they all work but my other Datagrids suffer the problem you highlighted.

**Comment by bdryanovski** [comment]
It seems that the both issues are related to each other #3824 and #3486 

```html
<clr-dg-column [clrDgField]="'name'"> Name
  <clr-dg-string-filter [clrDgStringFilter]="myCustomFilter"></clr-dg-string-filter>
</clr-dg-column>
```

When `[clrDgField]="'name'"` is removed the filter starts working and filtering the content. Not sure yet what is the reason for that. Have to investigate more deeply.

**Comment by kathygit** [comment]
We shall close this as duplicate to #3486

**Comment by gnomeontherun** [comment]
This should be resolved by #3859, closing as resolved.
